bigbluebutton-html5/imports/ui/components/connection-status/modal/component.jsx

```diff
@@ -107,6 +111,8 @@ class ConnectionStatusComponent extends PureComponent {
       const itemStyle = {};
       itemStyle[styles.even] = (index + 1) % 2 === 0;
 
+      const textStyle = {};
+      textStyle[styles.offline] = conn.offline;
```

Comment:
I think this css class need to be reincluded.

Reply:
done!